packages/esm-patient-vitals-app/src/config-schema.ts

```diff
@@ -42,6 +42,10 @@ export const configSchema = {
       _type: Type.ConceptUuid,
       _default: '1343AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
     },
+    vitalsSignsUuid: {
```

Comment:
```suggestion
    vitalSignsConceptSetUuid: {
```

Because this property represents a concept set:

![CleanShot 2024-12-04 at 11  19 32@2x](https://github.com/user-attachments/assets/4d4886a0-50df-41f1-a91a-3230c147d73a)